script/setup.js

```diff
@@ -169,6 +169,9 @@ try {
 
 	await withSpinner(
 		async () => {
+			const user = JSON.parse((await $`gh api user`).stdout);
+			const userName = user.login;
+			await $`all-contributors add ${userName} code,content,doc,ideas,infra,maintenance,projectManagement,tool`;
```

Comment:
> prefer to only use the provided owner name without reaching out to gh CLI

```plaintext
    stderr: 'gh: To use GitHub CLI in a GitHub Actions workflow, set the GH_TOKEN environment variable. Example:\n' +
```

Hmm, yeah, this is a situation... here are some starting thoughts:
* If `gh api user` _is_ available, we'd want to use it
* Running `all-contributors add` is nice for validating that the user does in fact exist
* If `gh api user` _isn't_ available, it makes sense to me to fall back on the provided owner name

What do you think about having the setup script trying, in order...
1. `gh api user` & `all-contributors add`
2. Provided owner & `all-contributors add` - and logging a warning
3. Throwing an error

...where we hardcode a username that we can expect to exist as the provided owner? Maybe ... yours? 😄

Reply:
Thanks for the feedback, I applied the following changes, let me know what you think